src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -15,6 +15,11 @@ export const terminalConfiguration: IConfigurationNode = {
 	title: localize('terminalIntegratedConfigurationTitle', "Integrated Terminal"),
 	type: 'object',
 	properties: {
+		'terminal.integrated.sendToShell': {
```

Comment:
I wonder whether it should say something about keys, like `sendKeybindingsToShell`?